web-console/src/druid-models/metric-spec/metric-spec.tsx

```diff
@@ -129,6 +139,14 @@ export const METRIC_SPEC_FIELDS: Field<MetricSpec>[] = [
       'longMax',
       'doubleMax',
       'floatMax',
+      'longFirst',
+      'longLast',
+      'doubleFirst',
+      'doubleLast',
+      'floatFirst',
+      'floatLast',
+      'stringFirst',
+      'stringLast',
```

Comment:
Perhaps this change isn't required. `KNOWN_TYPES` already contains the first/last aggregations.